bindings/python/algorithm/expose-centroidal.cpp

```diff
@@ -13,6 +13,23 @@ namespace pinocchio
     void exposeCentroidal()
     {
       using namespace Eigen;
+
+      bp::def("computeCentroidalDynamics",
+              &computeCentroidalDynamics<double,0,JointCollectionDefaultTpl,VectorXd,VectorXd>,
+              bp::args("Model","Data",
+                       "Joint configuration q (size Model::nq)",
+                       "Joint velocity v (size Model::nv)"),
+              "Computes the Centroidal dynamics, a.k.a. the total momenta of the system expressed around the center of mass.",
+              bp::return_value_policy<bp::return_by_value>());
```

Comment:
There is no need of return by value here.

Reply:
Every function that I see is using return by value as the policy.
what would you prefer here?

Reply:
It is because they are returning a matrix. But maybe it is fine like that avoiding miss understanding by the user.